test/src/api/assertions/testAttributeContains.js

```diff
@@ -58,7 +58,7 @@ describe('assert.attributeContains', function () {
         assert.strictEqual(instance.hasFailure(), false);
         assert.strictEqual(instance.getValue(), 'http://google.org');
         assert.strictEqual(instance.getActual(), 'http://google.org');
-        assert.strictEqual(err.message, `Error while running "attributeContains" command: Testing if attribute 'href' of element <.test_element> doesn't contain 'google' in 5ms - expected "not contains 'google'" but got: "http://google.org" (${instance.elapsedTime}ms)`);
+        assert.match(err.message, new RegExp(`Testing if attribute 'href' of element <.test_element> doesn't contain 'google' in 5ms - expected "not contains 'google'" but got: "http://google.org" \\(${instance.elapsedTime}ms\\)`));
```

Comment:
Actually I changed the error message back to what it used to be and I think this change is not needed.

Reply:
Would it be good to use regex expressions as it would be more concrete for future changes?

Reply:
I don't think so. Using regex in this case makes it less strict and also adds a bit more complexity. We know what the message should be like so we should test for an exact match.

Reply:
But I'm afraid you need to make some more changes. We've added the namespace as well and the type of error in the message.

Reply:
Sure😀